packages/react-components/src/molecules/PeopleCard.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import css from '@emotion/css';
+import format from 'date-fns/format';
+import { UserResponse } from '@asap-hub/model';
+
+import { Card, Link, Headline2, Avatar, Caption } from '../atoms';
+import ProfilePersonalText from './ProfilePersonalText';
+import { tabletScreen } from '../pixels';
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'row',
+  flexWrap: 'wrap',
+  justifyContent: 'space-between',
+  alignItems: 'start',
+});
+
+const textContainerStyles = css({
+  flexGrow: 1,
+  display: 'flex',
+  flexWrap: 'wrap',
+  justifyContent: 'space-between',
+  alignItems: 'start',
+  '> *': {
```

Comment:
This invades `ProfilePersonalText`'s styling and should IMO be avoided. We can wrap it a container that ensures it is big enough and then gives that space to `ProfilePersonalText`.